pepper-apis/studybuilder-cli/studies/pancan/subs.conf

```diff
@@ -177,7 +177,7 @@
         )
         ||
         (user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_CHILD"].numChildAnswers("PRIMARY_CANCER_CHILD") > 1
-          && user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_CHILD"].children["PRIMARY_CANCER_CHILD"].answers.hasOptionStartsWith("C_SARCOMAS_ANGIOSARCOMA", "C_GASTRO_ESOPHAGEAL_CANCER", "C_GASTRO_GASTRIC_STOMACH_CANCER", "C_GENITOURINARY_PROSTATE", "C_BREAST_")
```

Comment:
This file changes are just to keep config in sync with what patch is doing and patch doest use this file.